src/modules/pmc/src/lib/ExecutionState.h

```diff
@@ -6,34 +6,36 @@ class ExecutionState
 public:
     enum StateComponent
     {
-        unknown = 0,
-        running,
-        succeeded,
-        failed,
-        timedOut
+        Unknown = 0,
+        Running,
+        Succeeded,
+        Failed,
+        TimedOut
     };
 
     enum SubStateComponent
     {
-        none,
-        deserializingJsonPayload,
-        deserializingDesiredState,
-        deserializingSources,
-        deserializingPackages,
-        modifyingSources,
-        updatingPackageLists,
-        installingPackages
+        None,
+        DeserializingJsonPayload,
+        DeserializingDesiredState,
+        DeserializingGpgKeys,
+        DeserializingSources,
+        DeserializingPackages,
+        DownloadingGpgKeys,
+        ModifyingSources,
+        UpdatingPackageLists,
+        InstallingPackages
     };
 
     ExecutionState();
     virtual ~ExecutionState() = default;
 
     void SetExecutionState(StateComponent stateComponent, SubStateComponent subStateComponent, std::string processingArgument);
     void SetExecutionState(StateComponent stateComponent, SubStateComponent subStateComponent);
-    bool IsSuccessful();
-    StateComponent GetExecutionState();
-    SubStateComponent GetExecutionSubState();
-    std::string GetExecutionSubStateDetails();
+    bool IsSuccessful() const;
+    StateComponent GetExecutionState() const;
+    SubStateComponent GetExecutionSubState() const;
+    std::string GetExecutionSubStateDetails() const;

```

Comment:
substate is a noun. Everywhere we can have "Substate" or "substate" instead of "SubState" and respectively "subState".

Reply:
done!